src/components/DappBrowser/ErrorPage.tsx

```diff
@@ -0,0 +1,52 @@
+import lang from 'i18n-js';
+import React from 'react';
+import { Centered } from '../layout';
+import { Text } from '../text';
```

Comment:
you're playing with fire

Reply:
I heard you're great at making new screens